openedx/core/djangoapps/user_authn/views/auth.py

```diff
@@ -0,0 +1,9 @@
+from common.djangoapps.util.json_request import JsonResponse
+from django.conf import settings
+
+def get_public_signing_jwks():
```

Comment:
1. Reminder to fix quality issues in failing tests.
2. This should have a docstring. Ex:
```
"""
View to provide the auth related public signing JWK set as json.
"""
```